adam/language_specific/english/english_language_generator.py

```diff
@@ -228,12 +228,24 @@ def _noun_for_object(
                     f"an ontology node currently: {_object}"
                 )
             # TODO: we don't currently translate modifiers of nouns.
```

Comment:
As this issue has been resolved I think we can also remove this TODO.

Reply:
Removed